lib/chef/knife/bootstrap/bootstrapper.rb

```diff
@@ -16,7 +16,11 @@
 # limitations under the License.
 #
 
-require_relative "../../../azure/resource_management/ARM_interface"
+module Azure
+  class ResourceManagement
+    autoload :ARMInterface, File.expand_path("../../../azure/resource_management/ARM_interface", __dir__)
```

Comment:
```suggestion
    autoload :ARMInterface, "azure/resource_management/ARM_interface"
```

If this gem's `lib/` is in `$LOAD_PATH` this should be equivalent.